### DISCUSSION TITLE
GitHub Discord Server

### DISCUSSION THREAD

**Discussion by doublevcodes** [community/community#23596] [New to GitHub]
<p>Hey guys,<br>
Recently I was in search of a Discord server focused around GitHub and open-source development, but failed to do so.<br>
Therefore, I took it upon myself to create a Discord server about the aforementioned topic and thought this would be a nice place to share for those looking for a support forum with a more real-time nature.<br>
You can find an invite link to the Discord server within the organisation README of the community’s GitHub organisation (<a href="https://github.com/gh-discord" rel="noopener nofollow ugc">https://github.com/gh-discord</a>).</p>
<p>I hope to see some of you in the Discord server where I hope you’ll have a great time.</p>
<p>P.S. if this goes against the rules in anyway, please politely let me know and I’ll remove the post. Thanks guys!</p>

**Comment by airtower-luna** [comment]
<p>I can’t speak for GitHub, but I think you might want to make it clear that it’s a user group, not an official GitHub thing, see also: <a href="https://github.com/logos" class="inline-onebox">GitHub Logos and Usage · GitHub</a></p>
<p>Have fun! :smiley_cat:</p>

**Comment by doublevcodes** [comment]
<p>Right, I’ve emailed GitHub about this and haven’t received a reply for a few weeks, so I thought I’d just go with it. Any suggestions on how to make it more “unofficial” would be appreciated. Thanks!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Most importantly, don’t use the GitHub logo as yours. :wink:</p>
<p>And also make it clear in the language. If you say “GitHub Discord” it sound like it’s something <em>by</em> GitHub. “User group” <a href="https://en.wikipedia.org/wiki/User_group">is a traditional term</a> to describe groups for users of a particular thing, without speaking for the thing in general. So maybe you could use GUG (GitHub User Group), or make up something with similar meaning.</p>

**Reply by srijon57** [reply]
❤️

**Comment by doublevcodes** [comment]
<p>Alright, plans:</p>
<ol>
<li>Make a better custom logo</li>
<li>Change the name, I’m thinking something like “Git and GitHub”<br>
Thanks for the advice :grin:
</li>
</ol>

**Comment by naikrovek** [comment]
<p>I get a 404 clicking that link.  Is already abandoned?  GitHub needs an official discord or slack presence, imo, but i’ll take an unofficial one if that’s all that’s available.</p>

**Comment by nh916** [comment]
whats the discord server link?

I am looking for a github discord server

**Comment by 0xAskar** [comment]
The link throws a 404 error, any updated link?

**Reply by doublevcodes** [reply]
Sorry: https://discord.gg/pabeWKhqPt

**Comment by doublevcodes** [comment]
A newer link that should work is https://discord.gg/pabeWKhqPt

**Comment by nikhilsohan** [comment]
link doesn't work

**Reply by doublevcodes** [reply]
> link doesn't work 

https://discord.gg/pabeWKhqPt try this

**Reply by VISHNUKUMARA2010** [reply]
Link doesn't work

**Comment by ak-0283** [comment]
I am facing a problem i.e I have created a folder name project, under project I have created project 1 folder now I waana host that project 1 folder not that project folder. How to solve this problem.

**Comment by NothingButTyler** [comment]
The invite link **does not** work.

**Comment by miketopeak** [comment]
https://discord.com/invite/89eSFd3hdn
it will works on your side.

**Comment by deve-yunush-1** [comment]
<html>
<body>

Digest: sha256:78c967bda0083e6d17e7283356111302e708762243ac0496695b1180c3ca1696
--
391 | Status: Downloaded newer image for gcr.io/buildpacks/gcp/run:v1
392 | ===> ANALYZING
393 | Image with name "us-central1-docker.pkg.dev/alpha-service-453510/cloud-run-source-deploy/nipstec-connect-service/nipstec-connect-service:56641e411757d2322ee46e08c8f58af21fb8bc7d" not found
394 | ===> DETECTING
395 | target distro name/version labels not found, reading /etc/os-release file
396 | 4 of 5 buildpacks participating
397 | google.java.runtime      0.9.2
398 | google.java.gradle       0.10.0
399 | google.config.entrypoint 0.9.0
400 | google.utils.label-image 0.0.2
401 | ===> RESTORING
402 | ===> BUILDING
403 | target distro name/version labels not found, reading /etc/os-release file
404 | === Java - Runtime (google.java.runtime@0.9.2) ===
405 | Using latest Java 11 runtime version. You can specify a different version with GOOGLE_RUNTIME_VERSION: https://github.com/GoogleCloudPlatform/buildpacks#configuration
406 | 2025/03/13 05:43:05 [DEBUG] GET https://dl.google.com/runtimes/ubuntu1804/openjdk/version.json
407 | ***** CACHE MISS: "openjdk"
408 | Installing  v11.0.26+4.
409 | 2025/03/13 05:43:05 [DEBUG] GET https://dl.google.com/runtimes/ubuntu1804/openjdk/openjdk-11.0.26_4.tar.gz
410 | === Java - Gradle (google.java.gradle@0.10.0) ===
411 | --------------------------------------------------------------------------------
412 | Running "./gradlew clean assemble -x test --build-cache --quiet"
413 | Done "./gradlew clean assemble -x test --build-cache --quiet" (1.070469ms)
414 | --------------------------------------------------------------------------------
415 | failed to build: (error ID: 14f2a5b3):
416 | executing command "./gradlew clean assemble -x test --build-cache --quiet": fork/exec ./gradlew: permission denied
417 | --------------------------------------------------------------------------------
418 | Sorry your project couldn't be built.
419 | Our documentation explains ways to configure Buildpacks to better recognise your project:
420 | -> https://cloud.google.com/docs/buildpacks/overview
421 | If you think you've found an issue, please report it:
422 | -> https://github.com/GoogleCloudPlatform/buildpacks/issues/new
423 | --------------------------------------------------------------------------------
424 | ERROR: failed to build: exit status 1
425 | ERROR: failed to build: executing lifecycle: failed with status code: 51
426 | Finished Step #0 - "Buildpack"
427 | ERROR
428 | ERROR: build step 0 "gcr.io/k8s-skaffold/pack" failed: step exited with non-zero status: 1


</body>
</html>  I am getting this error while deploying on google cloude run. I don`t understood how can I resolve this issue can anyone help me to find this exactly what kind of issue

**Comment by serikan885** [comment]
彜簈r`�?�4$�aH琱Я�,�;顲抝

**Reply by serikan885** [reply]
Abi bu nedir ya nasıl çözülür

**Comment by nicenic-maintainer** [comment]
ICANN Accredited Registrar: https://discord.gg/VvSYcwPM

**Comment by WesleyBarta** [comment]
hi my name is Jack and I am finding collaboration people to hire

**Comment by v-coder247** [comment]
Hi, all.

Thanks for being part of the community!

Kindly note that you can now join the official GitHub Next discord server: https://gh.io/next-discord

I will proceed to closed this discussion.
